javascripts/govuk/show-hide-content.js

```diff
@@ -0,0 +1,150 @@
+(function (global) {
+  'use strict';
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+
+  function ShowHideContent() {
+    var self = this;
+
+    // Radio and Checkbox selectors
+    var selectors = {
+      namespace: 'ShowHideContent',
+      radio: '.block-label input[type="radio"]',
+      checkbox: '.block-label input[type="checkbox"]'
```

Comment:
We might want make sure we're not using presentational classes for binding, this will make this module more flexible.

(https://github.com/alphagov/styleguides/blob/master/js.md#html-class-hooks)

For example

```html
<label class="block-label js-toggle-showhide" data-toggle-target="show-me">
  <input type="radio" name="enabled" value="yes" /> Yes
</label>

<div id="show-me" class="panel js-hidden">
  <p>Show/Hide content to be toggled</p>
</div>
```